### PR TITLE
Installing juju

### LINKED ISSUE #15: Add Juju install/setup info
- [x] Add structure
 - [ ] Add navigation
 - [x] Intro
 - [x] Scope and Assumptions
 - [x] Install
 - [x] Configure for MAAS
 - [x] Testing
 - [x] Further reading/references

### PR DESCRIPTION
**Pull request by degville**
Fixes #15 

First draft of the installing Juju doc for the new cloud documentation.

### CODE REVIEW

File: en/install-juju.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+In the [previous step][installmaas], we installed, deployed and configured
+[MAAS][maas] to use as a foundation for Juju to deploy a fully fledged
+OpenStack cloud. 
+
+We're now going to install and configure the following two core components of
+Juju to use our MAAS deployment:
+
+- The *controller* is the management node for a cloud environment. We'll be
+  using the MAAS node we tagged with `juju` to contain and our Juju controller.
```
Comment (evilnick) [inline en/install-juju.md]: contain and operate? something is missing anyhow

File: en/install-juju.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+We're going to start by installing the Juju client on a machine running [Ubuntu
+16.04][xenialdownload] LTS (Xenial) with network access to the MAAS deployment.
+For other installation options, see [Getting started with Juju][otherinstall].
+
+To install Juju, enter the following in the terminal:
+
+```bash
+sudo add-apt-repository -u ppa:juju/stable
+sudo apt install juju
```
Comment (evilnick) [inline en/install-juju.md:32]: apt update?

**Review by evilnick — CHANGES_REQUESTED** [review]
nice!

File: en/install-juju.md (hunk 2 above)
Comment (degville) [reply en/install-juju.md:32]: that's the `-u`

File: en/install-juju.md (hunk 1 above)
Comment (degville) [reply en/install-juju.md]: thanks. I'll change this.

File: en/install-juju.md (hunk 2 above)
Comment (evilnick) [reply en/install-juju.md:32]: oh yes, sorry, I missed that

**Review by evilnick — APPROVED** [review]

**Pull request merged**